### ISSUE TITLE
UtilityCost:Ratchet code doesn't match documentation

### ISSUE THREAD

**Issue by mwilsoncea** [NREL/EnergyPlus#8854]
The Input Ouput Reference for UtilityCost:Ratchet states if the Multiplier Value is left blank a value of 1 will be used:

"3.8.1.0.7 Field: Multiplier Value or Variable Name
Often the ratchet has a clause such as “the current month demand or 90% of the summer month
demand”. For this case a value of 0.9 would be entered here as the multiplier. This value may be left blank
if no multiplier is needed and a value of **one will be used as a default.**"

A run of the example file 5ZoneEconomicsTariffAndLifecycleCost.idf with the multiplier set to 1 and left blank yields different results for the utility rate "ExampleDWithRatchet" indicating that 1 is not the default.

The actual code indicates a value of 0 will be used as default:

https://github.com/NREL/EnergyPlus/blob/e0d2dea7123012ea6f5bdea4cb007f1f01d70993/src/EnergyPlus/EconomicTariff.hh#L565-L574

Please update the documentation or code.

**Comment by jmarrec** [comment]
This is a documentation error. If the default value was one, it'll always enforce `min(currentMonthDemand, SummerDemand)`. #8902 will fix it.

**Comment by mjwitte** [comment]
> This is a documentation error. If the default value was one, it'll always enforce min(currentMonthDemand, SummerDemand).

I disagree. The value should default to 1.0, otherwise, the Ratchet is doing nothing more than passing the Baseline Source Variable value (TotalDemand in this case). @JasonGlazer Please confirm. This impacts a doc change in #9057.

**Comment by JasonGlazer** [comment]
The code does not show that it uses 1 by default and that is why you are seeing diffs. It is very old code and it is possible that it used to have a default value of 1 but changing it to C++ or some subsequent refactoring got rid of that. That input should really not be getting defaulted since it is so fundamental to what a ratchet performs. I think defaulting it to 1 makes more sense than 0 since 0 means the ratchet is really not doing anything at all.

**Comment by RKStrand** [comment]
Since this requires a change to the IDD and possibly code changes as well, this will have to wait until after the release.  I will back out the changes I made for #8902.

**Comment by mjwitte** [comment]
@RKStrand Any plan to make the proposed IDD change for the upcoming 22.2 I/O Freeze?
